tutorials/developers/dev_nn_cla/dev_nn_cla.tex

```diff
@@ -0,0 +1,395 @@
+\documentclass{tufte-handout}
+\usepackage{../braph2_dev}
+%\geometry{showframe} % display margins for debugging page layout
+
+\title{Implement a new Neural Network Classifier}
+
+\author[The BRAPH~2 Developers]{The BRAPH~2 Developers}
+
+\begin{document}
+
+\maketitle
+
+\begin{abstract}
+\noindent
+This is the developer tutorial for implementing a new neural network classifier. 
+In this Tutorial, we will explain how to create the generator file \fn{*.gen.m} for a new neural network classifier, which can then be compiled by \code{braph2genesis}. All kinds of neural network models are (direct or indirect) extensions of the base element \code{NNBase}. Here, we will use as examples the neural network classifier \code{NNClassifierMLP} (multi-layer perceptron classifier).
+\end{abstract}
+
+\tableofcontents
+
+%%%%% %%%%% %%%%% %%%%% %%%%%
+\clearpage
+\section{Implementation of a Neural Network Classifier}
+
+We will start by implementing in detail \code{NNClassifierMLP}, which is a direct extension of \code{NNBase}.
+A multi-layer perceptron classifier \code{NNClassifierMLP} comprises a multi-layer perceptron classifier model and a given dataset.
+
+\begin{lstlisting}[
+	label=cd:m:NNClassifierMLP:header,
+	caption={
+		{\bf NNClassifierMLP element header.}
+		The \code{header} section of the generator code for \fn{\_NNClassifierMLP.gen.m} provides the general information about the \code{NNClassifierMLP} element.
+		}
+]
+%% ¡header!
+NNClassifierMLP < NNBase (nn, multi-layer perceptron classifier) comprises a multi-layer perceptron classifier model and a given dataset.  ¥\circled{1}\circlednote{1}{defines \code{NNClassifierMLP} as a subclass of \code{NNBase}. The moniker will be \code{nn}.}¥
+
+%%% ¡description!
+A neural network multi-layer perceptron classifier (NNClassifierMLP) comprises a multi-layer perceptron classifier model and a given dataset.
+NNClassifierMLP trains the multi-layer perceptron classifier with a formatted inputs ("CB", channel and batch) derived from the given dataset.
```

Comment:
Maybe take out the change of line between "...a given dataset." and "NNClassifierMLP trains..." , like it is in DESCRIPTION (constant, string)

Reply:
<img width="368" alt="image" src="https://github.com/giovannivolpe/BRAPH-2-Matlab-beta/assets/30530538/b710fd67-5d8e-4778-9662-2f683d522f2a">